src/insertion_sort.py

```diff
@@ -0,0 +1,56 @@
+# Similar to Uno card game - shuffling cards in my hand - swapping positions
+
+# my_list = [8, 2, 5, 4, 1, 3]
+
+# my_list = [8, | 2, 5, 4, 1, 3]
+
+# temp = 2
+# my_list = [2, 8, | 5, 4, 1, 3]
+
+# temp = 5
+# my_list = [2, 5, 8, | 4, 1, 3]
+
+# temp = 4
+# my_list = [2, 4, 5, | 8, 1, 3]
+
+"""
+[8, 2, 5, 4, 1, 3]
+^
+[2, 5, 4, 1, 3] < [8]
+^
+[1] < [2] > [5, 4, 3][8]
+^
+[1][2][4, 3] < [5] > [8]
+^
+[1][2][3] < [4] > [5][8]
+
+"""
+
```

Comment:
Really love what you're doing here, diagramming and stepping through the code visually in comments- while it may be a bit more touchy than regular comments, visual explanations of the code + verbal ones will be super useful to you and you teammates.